ingest-app/src/cmr/ingest/config.clj

```diff
@@ -7,6 +7,10 @@
    [cmr.oracle.config :as oracle-config]
    [cmr.oracle.connection :as conn]))
 
+(defconfig progressive-update-enabled
+  "Flag for whether or not collection progressive update is enabled."
+  {:default true :type Boolean})
```

Comment:
Can you make a note in the deployment repo instructions that this flag exists now?

Reply:
Sure.